run_tests.sh

```diff
@@ -27,6 +27,8 @@ elif [ -e ${projectFolder}/.swift-test-macOS ] && [ "$osName" == "osx" ]; then
 elif [ -e ${projectFolder}/.swift-test-linux ] && [ "$osName" == "linux" ]; then
   echo ">> Running custom Linux test command: $(cat ${projectFolder}/.swift-test-linux)"
   source ${projectFolder}/.swift-test-linux
+elif [ -n "${RUN_TESTS_PARALLEL}"]; then
+  swift test --parallel
```

Comment:
An alternative would be to define `SWIFT_TEST_FLAGS` and always pass that as args to swift test.  Then in your Travis you could define `ENV SWIFT_TEST_FLAGS=--parallel` which I think has two advantages.  It's clearer what flags are being used, and if we want to specify additional ones in the future, we don't need further changes.